### DISCUSSION TITLE
File Manager Integration

### DISCUSSION THREAD

**Discussion by thabaum** [ADefWebserver/ADefWebserver.Module.HtmlTextV2#23] [General]
I am curious about the file manager.

I can put it on the top of the designer, or in the dialog popup that currently happens, both options when you click the image icon.

I can make it show up now, I just do not have the logic ready to make it work... so for the POC maybe just ability to show it off?   Or are we going for a full working file manager version for the POC?

**Comment by ADefWebserver** [comment]
No need to make it work at all. I say remove the button. If someone needs it this would be a good opportunity to make a PR :)

**Comment by thabaum** [comment]
PR #24 should resolve this for now.  I will work more on this over the weekend and over the next couple weeks until the meeting so I might get around to this if I get past a few other issues with the styling.  The next issue will be getting the drop downs CSS to work again.  I can remove these until they are working as well... My copy and this repo both have issues in this area so we need to get a grip on it next and should be good to go for a demo I am guessing.

**Reply by thabaum** [reply]
Should I just remove all the toolbar items until they function?   Keep only the ones that work as of now?  Then options shown currently would look full functional until these bugs are worked out by whomever can produce a PR.

I can give us a couple day to work it out before doing so but maybe something to consider if we are unable to make it work anytime soon.

**Reply by ADefWebserver** [reply]
> Should I just remove all the toolbar items until they function?

Yes
